pom.xml

```diff
@@ -14,7 +14,7 @@
   </licenses>
 
   <properties>
-    <raml-module-builder.version>33.0.0</raml-module-builder.version>
+    <raml-module-builder.version>33.1.3</raml-module-builder.version>
     <vertx.version>4.1.0.CR1</vertx.version>
```

Comment:
Bumpiing RMB version from 33.0.0 to 33.1.3 requires to also bump the vertx version.

One other way is to add
```
    <dependencies>
      <dependency>
        <groupId>org.apache.logging.log4j</groupId>
        <artifactId>log4j-bom</artifactId>
        <version>2.17.0</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
```
to the `<dependencyManagement>` section.

Or you bump RMB from 33.0.0 to 33.0.4, but this results only in log4j 2.16.0, adding log4j-bom results in the latest log4j 2.17.0.

Reply:
done